README.md

```diff
@@ -1,376 +1,242 @@
-# Spectrum CSS
+<div align="right">
 
-Spectrum CSS provides a standard CSS implementation of the Spectrum design language for internal and 3rd party use on Adobe's platforms.
+![GitHub Actions build status on main](https://img.shields.io/github/actions/workflow/status/adobe/spectrum-css/production.yml?branch=main&style=for-the-badge&label=main) <br/> ![GitHub Actions build status on spectrum-two](https://img.shields.io/github/actions/workflow/status/adobe/spectrum-css/production.yml?branch=spectrum-two&style=for-the-badge&label=spectrum-two)
 
-1. [Show me a demo](#show-me-a-demo)
-1. [Where is the JavaScript?](#where-is-the-javascript)
-1. [Using Spectrum CSS](#using-spectrum-css)
-1. [Contributing](#contributing)
+<h1 align="center">Spectrum CSS</h1>
+<h3 align="center">A CSS-implementation of the Spectrum design language</h3>
 
-## Show me a demo
+<img src="assets/spectrum-css_illustration_desktop.png">
 
-Check out the [component viewer](http://opensource.adobe.com/spectrum-css/preview) for a demo of every component included in Spectrum CSS or spin it up locally with: `yarn start`. This preview is driven by [Storybook](https://storybook.js.org/docs/web-components/get-started/introduction), a development tool that enables rapid prototyping and test suite integration.
+## Features
 
-Alternatively, you can preview the [documentation site](http://opensource.adobe.com/spectrum-css/) or spin up the site locally with: `yarn dev`.
+* 📖 **Robust documentation**: Spectrum CSS is designed to be used in partnership with [Spectrum's detailed usage guidelines](https://spectrum.adobe.com/).
+* 🎨 **Flexible**: Our CSS is customizable, powerful, and designed to work with any JavaScript framework.
+* 🧪 **Rigorously tested**: These individually-versioned components have been vetted to be accessible and inclusive of global audiences.
+* 📱 **Multi-platform support**: We support evergreen browsers (minus one version) for scalability and flexibility.
 
-## Where is the JavaScript?
+<div align="center">
 
-Spectrum CSS is CSS-only, implementing only the interactivity that can be done with pure CSS. Thus, Spectrum CSS should only be used by implementations of Spectrum, or very simple applications that only need things like typography, checkboxes, text fields, etc.
+![Get started](https://img.shields.io/badge/Get%20started-F0F0F0?style=for-the-badge&logo=adobe&logoColor=%23FF0000&link=https%3A%2F%2Fopensource.adobe.com%2Fspectrum-css%2Fget-started.html) ![Storybook link](https://img.shields.io/badge/Storybook-F0F0F0?style=for-the-badge&logo=storybook&logoColor=%23FF4785&link=https%3A%2F%2F63f7908f19912748edc50f0d-fgxdotpxra.chromatic.com)<br/> ![Spectrum Web Components link](https://img.shields.io/badge/Spectrum%20Web%20Components-F0F0F0?style=for-the-badge&logo=adobe&logoColor=%23FF0000&link=https%3A%2F%2Fopensource.adobe.com%2Fspectrum-web-components)
```

Comment:
```suggestion
[![Get started](https://img.shields.io/badge/Get%20started-F0F0F0?style=for-the-badge&logo=adobe&logoColor=%23FF0000)](https://opensource.adobe.com/spectrum-css/get-started.html) [![Storybook](https://img.shields.io/badge/Storybook-F0F0F0?style=for-the-badge&logo=storybook&logoColor=%23FF4785)](https://opensource.adobe.com/spectrum-css/preview/)<br/> [![Spectrum Web Components](https://img.shields.io/badge/Spectrum%20Web%20Components-F0F0F0?style=for-the-badge&logo=adobe&logoColor=%23FF0000)](https://opensource.adobe.com/spectrum-web-components/)
```
I don't think the badge `link` param works for markdown? It just takes me to the image (viewing preview of the file in GitHub, if this is accurate). I think this should work. Just double-check that Storybook URL—I wasn't sure if the Chromatic link was intentional.